zulip_bots/setup.py

```diff
@@ -65,6 +65,7 @@
         'html2text',
         'lxml',
         'BeautifulSoup4',
+        'typing_extensions',
```

Comment:
Is the change to requirements.txt necessary?
Do we need a specific/minimum version of typing_extensions?

Reply:
Yeah, they aren't, noticed it later.